### ISSUE TITLE
Simulations are requiring real Prayer Points

### ISSUE THREAD

**Issue by BillyMailman** [mythridium/combat-simulator#38]
**Describe the bug**
Currently, if I try to toggle prayers on/off in the simulator, it's popping up the error that "You don't have enough Prayer Points", which is silly, because the simulator shouldn't at all care how many real points I have. But, more importantly, if I then actually simulate while out of prayer, the simulations act like I haven't chosen any. This means all the kill time stuff is wrong, but more importantly, it means I can't use the simulator to calculate Prayer Point usage per hour, in order to then go get at least enough points for the grind I'm planning, until I already have Prayer Points.

**To Reproduce**
1. Run out of Prayer Points
2. Open the simulator
3. Toggle prayers in the simulator
4. Run a simulation, and verify that prayers aren't being factored in.

**Expected behavior**
The simulator should simulate the scenario I gave it - one where I'm using the selected prayers.

**Save**
https://melvoridle.com/save/yH9mXnLI

Note that I'm not quite 100% out of prayer points, so stuff like Thick Skin will work, but Battleheart/Divination are what I plan on during the grind to 99 Magic, and they do exhibit the bug.

**Mod List**
Literally just this - anything else I was using isn't updated to 1.3 yet, so I've disabled it all.

**Comment by BillyMailman** [comment]
Update: After posting this, I just used up some of the bones I had stocked up, and it's still not happy with those two prayers - they won't simulate properly, still give the out-of-points warning, and I can even turn on _other_ prayers at the same time.

![image](https://github.com/mythridium/combat-simulator/assets/4186916/c2aca2ef-ecfb-495c-a437-7d15b4fb7184)

I ended up having to refresh the page and reload the game in order to get my now non-zero prayer points to get picked up. Once I did that, though, everything started working correctly.
